src/core/VarReplay.jl

```diff
@@ -36,11 +38,15 @@ export  VarName,
 ###########
 # VarName #
 ###########
-struct VarName
-  csym      ::    Symbol        # symbol generated in compilation time
-  sym       ::    Symbol        # variable symbol
-  indexing  ::    String        # indexing
-  counter   ::    Int           # counter of same {csym, uid}
+struct VarName{sym}
+    csym      ::    Symbol        # symbol generated in compilation time
```

Comment:
`sym` stores the original variable name, while `csym` stores a randomly generated name. If we remove `sym`, the original variable name is lost. Do you plan to add an alternative mechanism for storing original variable information?

Reply:
`sym` is now part of the type of the `VarName`, it is not removed.